_includes/v21.1/orchestration/start-cockroachdb-helm-secure.md

```diff
@@ -1,5 +1,9 @@
 {{site.data.alerts.callout_danger}}
-The CockroachDB Helm chart is undergoing maintenance for compatibility with Kubernetes versions 1.17 through 1.21 (the latest version as of this writing). No new feature development is currently planned. For new production and local deployments, we currently recommend using a manual configuration (**Configs** option). If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.17 through 1.22 (the latest version as of this writing). No new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```

Comment:
Not sure why we need to mention 1.17. I think we could just say "compatible with Kubernetes 1.22 and earlier."

Also, I'd add, "However, no new feature development...". That however seems like the main point of the warning.

Reply:
Seems like we should take another look at the Limitations section as well and make sure we're saying the same things there: https://www.cockroachlabs.com/docs/stable/deploy-cockroachdb-with-kubernetes.html#limitations